docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-normalize-data-model/index.md

```diff
@@ -16,22 +16,52 @@ import Badges from '@site/src/components/Badges';
 <Badges badgeType="Maintained"></Badges>&nbsp;
 <Badges badgeType="SPAL"></Badges>
 
-
 # Snowplow Normalize Package
-:::note
 
+:::note
 Normalize in this context means [database normalization](https://en.wikipedia.org/wiki/Database_normalization), as these models produce flatter data, not statistical normalization.
 :::
 
-**The package source code can be found in the [snowplow/dbt-snowplow-normalize repo](https://github.com/snowplow/dbt-snowplow-normalize ), and the docs for the [macro design are here](https://snowplow.github.io/dbt-snowplow-normalize/#/overview/snowplow_normalize ).**
+**The package source code can be found in the [snowplow/dbt-snowplow-normalize repo](https://github.com/snowplow/dbt-snowplow-normalize), and the docs for the [macro design are here](https://snowplow.github.io/dbt-snowplow-normalize/#/overview/snowplow_normalize).**
+
+## Package Configuration
```

Comment:
This whole block fits in the quick start guide rather, as an optional config that they might want to consider when setting this up, not a core component that we need to start our main package description with I think.

Reply:
Makes sense sry!